ui/src/app/views/workspace/component.ts

```diff
@@ -32,7 +35,22 @@ export class WorkspaceComponent implements OnInit {
     this.cohortsService.getCohortsInWorkspace('123', '123').subscribe(
           cohortsReceived => {
             this.cohortList = cohortsReceived;
+            this.CohortEditService.list().then(
+              cohorts => {
+                for (const coho of cohorts) {
+                  this.cohortList.push(coho);
+                }
+              }
+            );
           });
     this.currentUrl = this.router.url;
   }
+
+  addCohort(): void {
+    this.router.navigate([this.currentUrl + '/cohort/' + this.CohortEditService.add()]);
```

Comment:
Makes sense. I will work on separating out add from edit in that case, because they will need different functionality. (Even if they look the same to the viewer.)